### DISCUSSION TITLE
Use resources outside of the root directory

### DISCUSSION THREAD

**Discussion by Adam-Antios** [quarto-dev/quarto-cli#4361] [Q&A] [answered]
I have a folder called `publish` that contains a quarto website. I want the website to use files located at `../entries` folder (relative to the `publish` directory). In other words I want to use resources located one directory up relative to the Quarto root directory. 

I have tried to set:

```
project:
    type: website
    resources: ../entries
```

but to no avail. How can I do that?

**Comment by dragonstyle** [comment, marked as answer]
Right now, we don't support using resources outside of a website project's directory. We preserve the structure of the project folder when we move and deploy resources, so referencing files outside that project directory isn't supported at this point.
